app/helpers/view_helper.rb

```diff
@@ -11,6 +11,7 @@ module ViewHelper
     checkbox: 'Viral::Form::CheckboxComponent',
     datepicker: 'Viral::Form::DatepickerComponent',
     dialog: 'Viral::DialogComponent',
+    distance: 'Viral::DistanceOfTimeComponent',
```

Comment:
**thought:** I don't know if this is an appropriate name, as it actually implies to me something like the distance to an object.  Can we call it `time_ago` or something similar?  I hate naming things.

Reply:
Completed in 8a2cbfabe8f262620fcade7c5ea1b47dcc8b332a